### PR TITLE
Feature#16qrscreen

### LINKED ISSUE #16: QR screen
# REQUIREMENTS
We are going to do QR screen

# DESIGN
COVID ALERTS V2

# Acceptance Criteria
List all required acceptance criteria
- [ ] Nice UI
- [ ] Camera working
- [ ] Code fixed with prettier
- [ ] Navigation still working

### PR DESCRIPTION
**Pull request by mariam-rivera**
# Description
We done the QR screen. Camera for scanning QR is working successfully.
 
 Examples:

  ## Screenshots 

Capture
dfs

### CODE REVIEW

File: COVID19/src/screens/Camera.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import QRCodeScanner from 'react-native-qrcode-scanner';
+//import {RNCamera} from 'react-native-camera';
+import styles from '../utils/styles/cameraStyles/cameraStyles';
+import constans from '../utils/constans';
+import Icon from 'react-native-vector-icons/Ionicons';
+
+const onSuccess = (text) => {
+  console.log(text.bounds.origin);
+  console.log(text.data);
+  console.log(text.bounds.origin[0].x);
```
Comment (LeyaimMagma) [inline COVID19/src/screens/Camera.js:12]: Remove logs

**Review by LeyaimMagma — CHANGES_REQUESTED** [review]

File: COVID19/src/screens/Camera.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {View, Text} from 'react-native';
+import QRCodeScanner from 'react-native-qrcode-scanner';
+//import {RNCamera} from 'react-native-camera';
```
Comment (fernandoamz) [inline COVID19/src/screens/Camera.js:4]: If you don't need this import, remove it

File: COVID19/src/screens/Locations.js [hunk 3]
```diff
@@ -70,7 +70,9 @@ const Location = () => {
         />
       </View>
       <View>
-        <TouchableOpacity style={styles.buttonContainer}>
+        <TouchableOpacity
+          onPress={() => navigation.navigate('Camera')}
+          style={styles.buttonContainer}>
           <Icon size={60} color="#5A4DCC" name="pluscircleo" />
```
Comment (fernandoamz) [inline COVID19/src/screens/Locations.js:76]: You should use the colors from your constant file

**Review by fernandoamz — CHANGES_REQUESTED** [review]

File: COVID19/src/screens/Camera.js (hunk 1 above)
Comment (vmagana3) [reply COVID19/src/screens/Camera.js:12]: Logs removed!

File: COVID19/src/screens/Camera.js (hunk 2 above)
Comment (vmagana3) [reply COVID19/src/screens/Camera.js:4]: Import coment removed!

File: COVID19/src/screens/Locations.js (hunk 3 above)
Comment (vmagana3) [reply COVID19/src/screens/Locations.js]: Resolved!

**Review by fernandoamz — APPROVED** [review]

**Review by LeyaimMagma — APPROVED** [review]

**Review by LPMendoza — APPROVED** [review]
You have done a good advanced in your project !!

**Review by Joaquin-Niembro — APPROVED** [review]

**Pull request merged**